A/Arpack/build_tarballs.jl

```diff
@@ -2,102 +2,46 @@ using BinaryBuilder
-for sym in ${CZ_SYMBOLS[@]}; do
-    SYMBOL_DEFS+=("-Dc${sym}=c${sym}_64" "-Dz${sym}=z${sym}_64")
-done
-
-# Add one-off symbol mappings; things that don't fit into any other bucket:
-for sym in scnrm2 dznrm2 csscal zdscal dgetrf dgetrs; do
-    SYMBOL_DEFS+=("-D${sym}=${sym}_64")
-done
-
-# Set up not only lowercase symbol remappings, but uppercase as well:
-SYMBOL_DEFS+=(${SYMBOL_DEFS[@]^^})
-
-FFLAGS="${FFLAGS} -O3 -fPIE -ffixed-line-length-none -fno-optimize-sibling-calls -cpp"
-BLAS=blastrampoline
-LAPACK=blastrampoline
-if [[ ${nbits} == 64 ]]; then
-    FFLAGS="${FFLAGS} -fdefault-integer-8 ${SYMBOL_DEFS[@]}"
+if [[ "${target}" == *-mingw* ]]; then
+    LBT=blastrampoline-5
+else
+    LBT=blastrampoline
 fi
 
 mkdir build
 cd build
-export LDFLAGS="${EXE_LINK_FLAGS[@]} -L${libdir} -lpthread"
-cmake .. -DCMAKE_INSTALL_PREFIX="$prefix" \
-    -DCMAKE_TOOLCHAIN_FILE="${CMAKE_TARGET_TOOLCHAIN}" -DCMAKE_BUILD_TYPE=Release \
-    -DEXAMPLES=OFF \
-    -DBUILD_SHARED_LIBS=ON \
-    -DBLAS_LIBRARIES="-l${BLAS}" \
-    -DLAPACK_LIBRARIES="-l${LAPACK}" \
-    -DCMAKE_Fortran_FLAGS="${FFLAGS}"
-
-make -j${nproc} VERBOSE=1
-make install VERBOSE=1
 
-# For now, we'll have to adjust the name of the Lbt library on macOS and FreeBSD.
-# Eventually, this should be fixed upstream
-if [[ ${target} == *-apple-* ]] || [[ ${target} == *freebsd* ]]; then
-    echo "-- Modifying library name for Lbt"
-
-    for nm in libarpack; do
-        # Figure out what version it probably latched on to:
-        if [[ ${target} == *-apple-* ]]; then
-            LBT_LINK=$(otool -L ${libdir}/${nm}.dylib | grep lib${BLAS} | awk '{ print $1 }')
-            install_name_tool -change ${LBT_LINK} @rpath/lib${BLAS}.dylib ${libdir}/${nm}.dylib
-        elif [[ ${target} == *freebsd* ]]; then
-            LBT_LINK=$(readelf -d ${libdir}/${nm}.so | grep lib${BLAS} | sed -e 's/.*\[\(.*\)\].*/\1/')
-            patchelf --replace-needed ${LBT_LINK} lib${BLAS}.so ${libdir}/${nm}.so
-        elif [[ ${target} == *linux* ]]; then
-            LBT_LINK=$(readelf -d ${libdir}/${nm}.so | grep lib${BLAS} | sed -e 's/.*\[\(.*\)\].*/\1/')
-            patchelf --replace-needed ${LBT_LINK} lib${BLAS}.so ${libdir}/${nm}.so
-        fi
-    done
+if [[ ${nbits} == 64 ]]; then
+    cmake .. -DCMAKE_INSTALL_PREFIX="$prefix" \
+        -DCMAKE_TOOLCHAIN_FILE="${CMAKE_TARGET_TOOLCHAIN}" -DCMAKE_BUILD_TYPE=Release \
+        -DBLAS_LIBRARIES=${LBT} -DBLAS_LINKER_FLAGS="-L${libdir}" \
+        -DLAPACK_LIBRARIES=${LBT} -DLAPACK_LINKER_FLAGS="-L${libdir}" \
+        -DEXAMPLES=OFF \
+        -DBUILD_SHARED_LIBS=ON \
+        -DINTERFACE64=1 -DSYMBOLSUFFIX=_64
+else
+    cmake .. -DCMAKE_INSTALL_PREFIX="$prefix" \
+        -DCMAKE_TOOLCHAIN_FILE="${CMAKE_TARGET_TOOLCHAIN}" -DCMAKE_BUILD_TYPE=Release \
+        -DBLAS_LIBRARIES=${LBT} -DBLAS_LINKER_FLAGS="-L${libdir}" \
+        -DLAPACK_LIBRARIES=${LBT} -DLAPACK_LINKER_FLAGS="-L${libdir}" \
+        -DEXAMPLES=OFF \
+        -DBUILD_SHARED_LIBS=ON
 fi
```

Comment:
This greatly reduces duplication
```suggestion
if [[ ${nbits} == 64 ]]; then
    FLAGS=(-DINTERFACE64=1 -DSYMBOLSUFFIX=_64)
fi
cmake .. -DCMAKE_INSTALL_PREFIX="$prefix" \
    -DCMAKE_TOOLCHAIN_FILE="${CMAKE_TARGET_TOOLCHAIN}" -DCMAKE_BUILD_TYPE=Release \
    -DBLAS_LIBRARIES=${LBT} -DBLAS_LINKER_FLAGS="-L${libdir}" \
    -DLAPACK_LIBRARIES=${LBT} -DLAPACK_LINKER_FLAGS="-L${libdir}" \
    -DEXAMPLES=OFF \
    -DBUILD_SHARED_LIBS=ON \
    "${FLAGS[@]}"
```

Reply:
Thanks. I was just being lazy since I couldn't remember how to do it.